spec/factories.rb

```diff
@@ -54,6 +54,14 @@
     end
   end
 
+  factory :step_by_step_page_with_secondary_content_and_navigation_rules, parent: :step_by_step_page_with_steps do
```

Comment:
I'm not why this factory has been added? Could the existing `step_by_step_page_with_navigation_rules` be used instead?

Reply:
`step_by_step_page_with_navigation_rules` doesn't have secondary content links, which we also need in this test